src/cow-react/modules/trade/containers/ImportTokenModal/index.tsx

```diff
@@ -16,11 +16,16 @@ export function ImportTokenModal(props: ImportTokenModalProps) {
   const { chainId, onDismiss } = props
 
   const tradeState = useTradeState()
-  // token warning stuff
-  const [loadedInputCurrency, loadedOutputCurrency] = [
-    useCurrency(tradeState?.state?.inputCurrencyId),
-    useCurrency(tradeState?.state?.outputCurrencyId),
-  ]
+  const loadedInputCurrency = useSearchInactiveTokenLists(
+    tradeState?.state?.inputCurrencyId || undefined,
+    1,
+    true //
```

Comment:
U were going to write a comment here?

Reply:
No, it's to keep the line break. Without the comment prettier make it in one line (and it looks not so good)